packages/angular_devkit/build_angular/src/browser/specs/lazy-module_spec.ts

```diff
@@ -132,7 +132,7 @@ describe('Browser Builder lazy modules', () => {
       'src/main.ts': `import('./lazy-module');`,
     });
     // Using `import()` in TS require targetting `esnext` modules.
-    host.replaceInFile('src/tsconfig.app.json', `"module": "es2015"`, `"module": "esnext"`);
+    host.replaceInFile('src/tsconfig.app.json', `"module": "es2017"`, `"module": "esnext"`);
```

Comment:
This entire line shouldn't be needed anymore since new projects set the `module` option to `es2020`.
Same for the other changed line in this file below.